TYPO3.Neos/Classes/TYPO3/Neos/Command/SiteCommandController.php

```diff
@@ -65,6 +70,101 @@ class SiteCommandController extends CommandController
     protected $systemLogger;
 
     /**
+     * @Flow\Inject
+     * @var ContextFactoryInterface
+     */
+    protected $nodeContextFactory;
+
+    /**
+     * @Flow\Inject
+     * @var NodeTypeManager
+     */
+    protected $nodeTypeManager;
+
+    /**
+     * @Flow\Inject
+     * @var NodeService
+     */
+    protected $nodeService;
+
+    /**
+     * @Flow\Inject
+     * @var PersistenceManagerInterface
+     */
+    protected $persistenceManager;
+
+    /**
+     * Create a new (blank) site in the default dimension
+     *
```

Comment:
Nitpick: "Create a new (blank) site in the default dimension" is a bit long for the help screen, I'll change it to something shorter